### ISSUE TITLE
Content card fails WCAG 2.1 AA

### ISSUE THREAD

**Issue by mrmjprice** [wmcadigital/wmn-design-system#549]
**Describe the bug**
No hover state for clickable content cards

Needs to be visibly different when the user is hovering over the card.

Example: https://service-manual.nhs.uk

**Comment by jkmnch** [comment]
We have decided that the Navigation and Yellow modifier content cards will now have an underline style in their H2 titles. This is to show that the cards link to another page and to distinguish themselves from Normal content cards. When a user hovers over the Navigation Card or Yellow modifier card, a 1px Primary Purple border will appear around the card (similar to NHS). We cannot change the colour of the text (like NHS does) to Secondary Purple in the hover state, as this does not suit the Yellow modifier cards.

We have left the H2 underline in hover state to indicate the H2 title is still clickable on hover.

Let me know what you think. 

![Screenshot 2021-01-04 at 10 32 32](https://user-images.githubusercontent.com/62507489/103526640-2002dd00-4e79-11eb-94e0-4209343ee46f.png)

**Comment by houbly** [comment]
This update is now live
